src/pybind/mgr/volumes/module.py

```diff
@@ -604,12 +623,18 @@ def handle_command(self, inbuf, cmd):
     def _cmd_fs_volume_create(self, inbuf, cmd):
         vol_id = cmd['name']
         placement = cmd.get('placement', '')
+        errno, out, err = self._are_names_valid(cmd['name'])
```

Comment:
Module `errno` has been imported in this file and that would be overriden by the variable `errno` created here. Although it is not a problem now but it can become a problem later. Better to use a different name. Perhaps `errnum` or `_errno`?

Reply:
replaced with `_errno`